src/components/assessments/AssessmentCard.tsx

```diff
@@ -1,73 +1,38 @@
-import {Box} from '@mui/material'
+import {Card, CardActions} from '@mui/material'
 import CardContent from '@mui/material/CardContent'
 import {styled} from '@mui/material/styles'
 import Typography from '@mui/material/Typography'
-import createStyles from '@mui/styles/createStyles'
-import makeStyles from '@mui/styles/makeStyles'
 import {FunctionComponent} from 'react'
-import {theme} from '../../style/theme'
+import {latoFont, theme} from '../../style/theme'
 import {Assessment} from '../../types/types'
 import AssessmentImage from './AssessmentImage'
 
-const CardContainer = styled(Box, {label: 'cardContainer'})(({theme}) => ({
-  backgroundColor: '#FFFFFF',
-
-  boxShadow: '0px 1px 4px rgba(0, 0, 0, 0.15)',
+const StyledCard = styled(Card)(({theme}) => ({
+  width: theme.spacing(28),
+  height: theme.spacing(47),
+  borderTop: `5px solid #8FD6FF`,
+  textAlign: 'left',
+  borderRadius: 0,
+  padding: 0,
+
+  display: 'flex',
+  flexDirection: 'column',
+  color: '#353A3F',
+  boxShadow: '0px 4px 4px rgba(0, 0, 0, 0.25)',
 }))
 
-const useStyles = makeStyles(theme =>
-  createStyles({
-    /* root: {
-      width: theme.spacing(28),
-      height: theme.spacing(47),
-      textAlign: 'left',
-      // border: '1px solid gray',
-      padding: 0,
-
-      display: 'flex',
-      flexDirection: 'column',
-    },*/
-    dragging: {
-      width: '250px !important',
-      height: '250px !important',
-    },
-
-    /* content: {
-      height: theme.spacing(19),
-      overflowY: 'scroll',
-      '&::-webkit-scrollbar': {
-        width: '6px',
-      },
-      '&::-webkit-scrollbar-thumb': {
-        backgroundColor: '#C4C4C4',
-      },
-      zIndex: 100,
-    },
-
-    title: {
-      fontFamily: poppinsFont,
-      fontSize: 14,
-      fontWeight: 'bold',
-    },
-    summary: {
-      flexGrow: 0,
-      fontSize: 12,
-
-      paddingBottom: 0,
-    },*/
-
-    /*  bottom: {
-      marginTop: 0,
-      display: 'flex',
-      fontFamily: playfairDisplayFont,
-      fontStyle: 'italic',
-      fontSize: '12px',
-      padding: theme.spacing(1, 2),
-      alignItems: 'center',
-      justifyContent: 'space-between',
-    },*/
-  })
-)
+const StyledCardContent = styled(CardContent)(({theme}) => ({
+  height: theme.spacing(46),
+  overflowY: 'scroll',
+  paddingBottom: 0,
+  '&::-webkit-scrollbar': {
+    width: '6px',
+  },
+  '&::-webkit-scrollbar-thumb': {
+    backgroundColor: '#C4C4C4',
+  },
```

Comment:
Does the scrollbar need to be restyled? If so, we may need to look into a cross-browser solution and consider whether the styling will be reliable. For example, the [MDN Web Docs](https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar) note the following about `::-webkit-scrollbar` -- 

> Non-standard: This feature is non-standard and is not on a standards track. Do not use it on production sites facing the Web: it will not work for every user. There may also be large incompatibilities between implementations and the behavior may change in the future.

Reply:
Added a TODO comment.